checkmk_extensions/agent/scheduler_runner.ps1

```diff
@@ -43,8 +43,13 @@ class Config {
     }
 
     static [Config] ParseConfigFile([string]$Path) {
-        $configFileContent = Get-Content -Raw -Path $Path
-        $configData = ConvertFrom-Json -InputObject $configFileContent
+        try {
```

Comment:
I don't think we need that. The line `$ErrorActionPreference = "Stop"` should basically take care of this. I thought I had tested this. Is this not the case?

Reply:
It will cause the script to stop executing, but we will not log the error. Simon wrote in the specification that we should log it, so this is the way to ensure that it will be logged.

Reply:
No need for change, since we will not log this error.